Lib/fontgoggles/font/baseFont.py

```diff
@@ -92,6 +93,23 @@ def axes(self):
             axes[axis.axisTag] = axisDict
         return axes
 
+    @cachedProperty
+    def instances(self):
+        fvar = self.ttFont.get("fvar")
+        if fvar is None:
+            return []
+        name = self.ttFont["name"]
+        instances = []
+        for i in fvar.instances:
+            try:
+                # Assuming VFs have ID 16 set; should this fallback to ID 1 or "" if not?
```

Comment:
What's your thoughts on this?

Reply:
That is to say, should a VF have instances but is missing ID 16 it's instances will be added as "None - Regular Etc.".

Reply:
Improved this to show _some_ family, always, as part of supporting designspaces.